packages/esm-patient-allergies-app/src/allergies/allergies-detailed-summary.component.tsx

```diff
@@ -74,7 +74,7 @@ const AllergiesDetailedSummary: React.FC<AllergiesDetailedSummaryProps> = ({ pat
           </span>
         ),
       },
-      recordedDate: formatDate(parseDate(allergy.recordedDate), { day: false, time: false }) ?? '--',
+      onsetDate: formatDate(parseDate(allergy.onsetDate), { day: false, time: false }) ?? '--',
```

Comment:
@gracepotma looking at the data being returned by the API, we do not have a field named `onsetDate` we have a `recordedDate` and `lastUpdated Date`. So if we want to show the `onSetDate` we will need the API to be modified so that it returns it. cc @ibacher @denniskigen

Reply:
<img width="560" alt="Screenshot 2023-03-27 at 01 15 36" src="https://user-images.githubusercontent.com/30952856/227808588-2687ec66-6752-49da-a258-93a22911ebac.png">
I have also looked at the fetched resource, I have not seen the `onSetDate`.

Reply:
Ah I see. Wow. I assumed onsetDate could be used as a reference because I saw it used elsewhere in the allergies app here: https://github.com/openmrs/openmrs-esm-patient-chart/blob/f5330c710fa4f6f6d75d0659e7d15a7d8887d683/packages/esm-patient-allergies-app/src/allergies/allergies-form/allergy-form.component.tsx#L55

Reply:
If there is no possible way to represent this in the data model, then I propose we just remove this column completely for now, as it is misleading.

Reply:
Okay I will push that change on this PR.